terraform/autospotting/variables.tf

```diff
@@ -47,7 +49,24 @@ variable "autospotting_regions_enabled" {
 }
 
 variable "autospotting_tag_filters" {
-  description = "tags to filter which ASGs autospotting considers.  by default if left blank, asgs tagged with spot-enbled=true will be operated on.  You can set this to many tags, for example: spot-enabled=true,Environment=dev,Team=vision"
+  description = <<EOF
+  Tags to filter which ASGs autospotting considers. By default, ASGs tagged
+  with spot-enbled=true will be operated on. In opt-out mode it operates on
+  all groups except those tagged with spot-enbled=false.
```

Comment:
I think that the tags for opt-out should be also an input; so that people could simply opt-out all `dev` ASGs for example. Otherwise, we might run in the same case previously established for `spot-enabled=true`

Reply:
Not sure I understand your comment.

By design this field should be used in both opt-in and opt-out mode and its behavior is only controlled by the autospotting_tag_filtering_mode variable.

Reply:
My understanding when I first read the PR was that the opt-out, would only consider `spot-enabled=false` and no other tags. For example, we might have wanted to opt-out all production environments ASGs.

Maybe I got it wrong, but I feel that it would have been useful, if that wasn't the case.

Reply:
It will consider all the tags configured in the tag filter list. But when none is configured it takes the `spot-enabled=false`